GitHub_LearningTests/Feature/Dashboard/Resource/CryptocurrencyPriceResourceTests.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  CryptocurrencyPriceResourceTests.swift
+//  GitHub_LearningTests
+//
+//  Created by Amin Ashhab on 28.12.2020.
+//  Copyright © 2020 Amin_Second_Test_Project. All rights reserved.
+//
+
+import XCTest
+import CleanCore
+@testable import GitHub_Learning
+
+class CryptocurrencyPriceResourceTests: XCTestCase {
+    private var sut: CryptocurrencyPriceResourceImpl!
+    
+    private func setupTest(
+        httpNetworkClient: HttpNetworkClient = HttpNetworkClientDummy(),
+        businessErrorParser: BusinessErrorParser = BusinessErrorParserDummy(),
+        requestCreationStep: RequestCreationStep = RequestCreationStepDummy(),
+        responseBodyDeserializationStrategy: ResponseBodyDeserializationStrategy = ResponseBodyDeserializationStrategyDummy(),
+        baseUrlConfig: BaseUrlConfig = BaseUrlConfigDummy(),
+        cryptocurrencyParser: CryptocurrencyParser = CryptocurrencyParserDummy()
+    ) {
+        self.sut = CryptocurrencyPriceResourceImpl(
+            networkClient: httpNetworkClient,
+            errorParser: businessErrorParser,
+            creationStep: requestCreationStep,
+            deserializationStrategy: responseBodyDeserializationStrategy,
+            baseUrlConfig: baseUrlConfig,
+            cryptocurrencyPriceParser: cryptocurrencyParser)
+    }
+    
+    func test_givenSuccesResponse_whenGetCryptocurrency_thenCryptocurrencyIsValid() {
+        // Given
+        let string = "Success"
+        let data = Data(string.utf8)
+        let successResponse = HttpNetworkResponse(status: 200, headers: ["":""], body: data)
```

Comment:
```suggestion
        let successResponse = HttpNetworkResponse(status: 200, headers: [:], body: data)
```
Pokud tam nemuzes posilat nil, tak prazdny dictionary se pise takle `[:]`